### PR TITLE
Show warning when replying as an invisible participant

### PR DESCRIPTION
**Pull request by TimWolla**
see https://www.woltlab.com/community/thread/301789-warnung-%C3%BCber-dem-editor-wenn-man-ein-unsichtbarer-konversationsteilnehmer-ist/

### CODE REVIEW

File: templates/conversationQuickReply.tpl [hunk 1]
```diff
@@ -2,10 +2,13 @@
 	<article class="message messageSidebarOrientation{$__wcf->getStyleHandler()->getStyle()->getVariable('messageSidebarOrientation')|ucfirst}{if $__wcf->getUserProfileHandler()->userOnlineGroupID} userOnlineGroupMarking{$__wcf->getUserProfileHandler()->userOnlineGroupID}{/if}">
 		{include file='messageSidebar' userProfile=$__wcf->getUserProfileHandler()->getUserProfile() isReply=true enableMicrodata=false}
 		
-		<div class="messageContent messageQuickReplyContent"{if $pageNo < $pages} data-placeholder="{lang}wcf.conversation.reply{/lang}"{/if}>
+		<div class="messageContent messageQuickReplyContent"{if $pageNo <p $pages} data-placeholder="{lang}wcf.conversation.reply{/lang}"{/if}>
```
Comment (dtdesign) [inline templates/conversationQuickReply.tpl:5]:
```suggestion
		<div class="messageContent messageQuickReplyContent"{if $pageNo < $pages} data-placeholder="{lang}wcf.conversation.reply{/lang}"{/if}>
```

File: language/de.xml [hunk 2]
```diff
@@ -131,6 +131,7 @@
...
 		<item name="wcf.conversation.message.editNote"><![CDATA[{$message->username} hat diese Nachricht {#$message->editCount} Mal editiert, zuletzt: {time time=$message->lastEditTime}.]]></item>
 		<item name="wcf.conversation.noParticipantsWarning"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du bist{else}Sie sind{/if} dabei, auf eine Konversation ohne weitere Teilnehmer zu antworten. Alle anderen Teilnehmer haben diese Konversation verlassen. Niemand wird {if LANGUAGE_USE_INFORMAL_VARIANT}deine{else}Ihre{/if} Nachricht lesen!]]></item>
+		<item name="wcf.conversation.invisibleParticipantWarning"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du bist{else}Sie sind{/if} als unsichtbarer Teilnehmer dabei, auf diese Konversation zu antworten. Eine Antwort wird {if LANGUAGE_USE_INFORMAL_VARIANT}deine{else}Ihre{/if} Teilnahme an dieser Konversation für jeden anderen Teilnehmer sichtbar machen und die Antwort für jeden Teilnehmer sichtbar sein.]]></item>
```
Comment (dtdesign) [inline language/de.xml:134]:
```suggestion
		<item name="wcf.conversation.invisibleParticipantWarning"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du nimmst{else}Sie nehmen{/if} gegenwärtig an dieser Konversation als unsichtbarer Teilnehmer teil. Sobald {if LANGUAGE_USE_INFORMAL_VARIANT}du eine Antwort verfasst{else}Sie eine Antwort verfassen{/if}, wird diese für alle sichtbar sein und {if LANGUAGE_USE_INFORMAL_VARIANT}du bist{else}Sie sind{/if} nicht nicht länger für andere Teilnehmer unsichtbar.]]></item>
```

File: language/en.xml [hunk 3]
```diff
@@ -131,6 +131,7 @@
 		<item name="wcf.conversation.error.mailboxIsFull"><![CDATA[You have reached your maximum conversations limit and cannot create new conversations.]]></item>
 		<item name="wcf.conversation.message.editNote"><![CDATA[{$message->username} edited this message {#$message->editCount} times, last: {time time=$message->lastEditTime}.]]></item>
 		<item name="wcf.conversation.noParticipantsWarning"><![CDATA[You are about to reply to a conversation without other participants, nobody is going to read your message!]]></item>
+		<item name="wcf.conversation.invisibleParticipantWarning"><![CDATA[You are about to reply to this conversation as an invisible participant. Sending a reply will make your participation visible to all other participants and the reply will be visible to all participants.]]></item>
```
Comment (dtdesign) [inline language/en.xml:134]:
```suggestion
		<item name="wcf.conversation.invisibleParticipantWarning"><![CDATA[You are currently an invisible participant of this conversation. Any message you write in this conversation will be visible to every participant and you will no longer be invisible to others.]]></item>
```

**Review by dtdesign — CHANGES_REQUESTED** [review]

File: templates/conversationQuickReply.tpl (hunk 1 above)
Comment (TimWolla) [reply templates/conversationQuickReply.tpl:5]: whoops.

Comment (TimWolla) [reply templates/conversationQuickReply.tpl]: To have it noted, this compiled just fine and resulted in:

```
<?php if ($this->v['pageNo'] < 'p$pages') { ?>
```
:facepalm:

**Pull request merged**